src/Core/src/Platform/iOS/MauiActivityIndicator.cs

```diff
@@ -8,18 +8,21 @@ namespace Microsoft.Maui.Platform
 {
 	public class MauiActivityIndicator : UIActivityIndicatorView, IUIViewLifeCycleEvents
 	{
-		IActivityIndicator? _virtualView;
+		readonly WeakReference<IActivityIndicator>? _virtualView;
```

Comment:
Ok, it appears that interfaces slip through the analyzer:

![image](https://github.com/dotnet/maui/assets/840039/ff3e8231-62f0-4942-8b0d-946c59d04b75)

Will fix in next version of analyzer.